.gitignore

```diff
@@ -51,3 +51,5 @@ out/
 *.hnir
 test-report.json
 dump.lsif
+
+generated
```

Comment:
Does this gitignore new additions to the `tests/snapshots/src/main/generated/` directory?

Reply:
Yes it does. Resolved now to only affect top level